util/compression.h

```diff
@@ -615,7 +615,11 @@ inline std::string CompressionTypeToString(CompressionType compression_type) {
     case kXpressCompression:
       return "Xpress";
     case kZSTD:
+#if defined(ZSTD_VERSION_STRING)
+      return "ZSTD (" ZSTD_VERSION_STRING ")";
```

Comment:
This is stored in the table properties and maybe other places; I think we should avoid changing it unless we absolutely must.